src/frontend/js/hooks/useDateFormat/index.tsx

```diff
@@ -29,7 +30,9 @@ export const DATETIME_FORMAT: FormatDateOptions = {
 const useDateFormat = (formatOptions: FormatDateOptions = {}) => {
   const intl = useIntl();
 
-  function formatDate(date: string | Date | number, options: FormatDateOptions = {}) {
+  function formatDate(date: Maybe<string | Date | number>, options: FormatDateOptions = {}) {
+    if (!date) return undefined;
```

Comment:
Not fan of inline if clauses, it often lead to bad readability, but that's just my opinion